Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -838,7 +838,7 @@ private Region ConstructRegion(AnalyzeContext context, FlexMatch regionFlexMatch
             {
                 // match should be within the original full string
                 // update lengthOffset to till end of regionFlexMatch
-                lengthOffset = indexOffset - regionFlexMatch.Length;
+                lengthOffset = (regionFlexMatch.Length - indexOffset) - regionFlexMatch.Length;
```

Comment:
>regionFlexMatch.Length - indexOffset) - regionFlexMatch.Length [](http://example.com/codeflow?start=32&length=62)

doing this isn't similar to -indexOffset? and is that correct?

Reply:
yes it equals -indexOffset.
I was about to keep it in same style of this line: length of substring - length of full string
https://github.com/microsoft/sarif-pattern-matcher/blob/f86c1b5836aa0ef8ade200a305df9ae392c7a436/Src/Sarif.PatternMatcher/SearchSkimmer.cs#L827